Makefile

```diff
@@ -90,3 +90,20 @@ vendor:
 .PHONY: vendor-update
 vendor-update:
 	$(BUILDER) dep ensure
+
+.PHONY: image
+image:
+	docker build -f docker/Dockerfile.server -t infobloxopen/server:v1.0 .
+	docker build -f docker/Dockerfile.gateway -t infobloxopen/gateway:v1.0 .
```

Comment:
We do not have infobloxopen hub, I expect it should be infoblox. Also I would prefer image name that is more informative, for example infoblox/contacts-server:v1.0 & infoblox/contacts-gateway:v1.0